wordpress-mobile/WordPress-iOS
# Prologue Carousel: update headlines

Fixes #15064 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/498

This PR updates the headlines with the new typeface, font color, and headline text.

**Not included**
- new images and views for each carousel page, which will change the positioning of the text

### To test:
1. `bundle exec pod install`
2. build and run
3. Log out if logged in
4. Swipe through each slide and confirm the strings match the headline text and the font color and typeface match [the designs](https://github.com/wordpress-mobile/WordPress-iOS/issues/15056). 

Headline text
1. Welcome to the world’s most popular website builder.
2. With this powerful editor you can post on the go.
3. See comments and notifications in real time.
4. Watch your audience grow with in-depth analytics.
5. Follow your favorite sites and discover new reads.

- [ ] Uses the New York typeface. 
- [ ] Uses a dark text color on light mode and a light text color on dark mode

### Screenshots
| Light mode | Dark mode |
| --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-10-07 at 14 08 12](https://user-images.githubusercontent.com/1062444/95389508-8e0b2880-08b9-11eb-82a4-f4aff8fa9cdf.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-10-07 at 14 09 01](https://user-images.githubusercontent.com/1062444/95389526-94010980-08b9-11eb-9771-86006ac32a0f.png) |

<img width="1832" alt="Screen Shot 2020-10-07 at 4 25 51 PM" src="https://user-images.githubusercontent.com/1062444/95389682-c9a5f280-08b9-11eb-8e68-b75050076bba.png">

PR submission checklist:
